### PR TITLE
Removed unused dtsi files, and adjusted DCM DTS

### PR DESCRIPTION
**Pull request by MikeLoder**
- Removed the GPIO reference as this is now in yocto
- Fixed the gpio definitions to match the xavier
- Moved the button GPIO pins to gpio-keys so they trigger keyboard
events.

Mike i included you as a reviewer so you are aware of the changes i am making. Please dont review on your holiday.

### CODE REVIEW

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 1]
```diff
@@ -142,67 +141,61 @@
...
         // BUT_CENTERn => GPIO18 => SOC_GPIO40 GPIO03 => P(Q,4) *
-       button_center {
-            status = "okay";
-            input;
-            gpios = <TEGRA_GPIO(Q, 4) GPIO_ACTIVE_HIGH>;
-            line-name = "BUTTON-CENTERn";
-        };
+		button_center {
+			label = "BUTTON-CENTERn";
+            gpios = <&tegra_main_gpio TEGRA194_MAIN_GPIO(Q, 4) GPIO_ACTIVE_HIGH>;
```
Comment (m3harvey) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:154]: inconsistent tabbing.

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 2]
```diff
@@ -142,67 +141,61 @@
...
 
         // BUT_UPn => GPIO33 => SOC_GPIO05  GPIO3 => P(P,5) *
         button_up {
-            status = "okay";
-            input;
-            gpios = <TEGRA_GPIO(P, 5) GPIO_ACTIVE_HIGH>;
-            line-name = "BUTTON-UPn";
+            label = "BUTTON-UPn";
+            gpios = <&tegra_main_gpio TEGRA194_MAIN_GPIO(P, 5) GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_UP>;
```
Comment (m3harvey) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:162]: Inconsistent tabbing.

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 3]
```diff
@@ -142,67 +141,61 @@
...
-
         // BUT_DOWNn => GPIO17 => SOC_GPIO21 GPIO3 => P(Q,1) *
         button_down {
-            status = "okay";
-            input;
-            gpios = <TEGRA_GPIO(Q, 1) GPIO_ACTIVE_HIGH>;
-            line-name = "BUTTON-DOWNn";
+            label = "BUTTON-DOWNn";
+            gpios = <&tegra_main_gpio TEGRA194_MAIN_GPIO(Q, 1) GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_DOWN>;
```
Comment (m3harvey) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:169]: Inconsistent tabbing.

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 4]
```diff
@@ -142,67 +141,61 @@
...
 
         // BUT_LEFTn => GPIO29 => SOC_GPIO31   GPIO3 => P(S,6) *
         button_left {
-            status = "okay";
-            input;
-            gpios = <TEGRA_GPIO(S, 6) GPIO_ACTIVE_HIGH>;
-            line-name = "BUTTON-LEFTn";
+            label = "BUTTON-LEFTn";
+            gpios = <&tegra_main_gpio TEGRA194_MAIN_GPIO(S, 6) GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_LEFT>;
```
Comment (m3harvey) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:176]: Last one.

Comment (MikeLoder) [reply nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts]: Converted all tabs to spaces, thanks.

**Comment by MikeLoder** [comment]
@m3harvey Can you please reapprove

**Pull request merged**